utilities/ordered-multimap.metta

```diff
@@ -53,7 +53,27 @@
 (= (MultiMap.removeAll $key NilMMap) NilMMap)
 (= (MultiMap.removeAll $key (ConsMMap ($curKey $curVal) $tail)) 
     (if (== $key $curKey) (MultiMap.removeAll $key $tail) (ConsMMap ($curKey $curVal) (MultiMap.removeAll $key $tail))))
-    
+
+
+;; Get the all the values from the map
+(: MultiMap.values (-> (MultiMap ($v $v)) Expression))
```

Comment:
You should follow the same convetion here as well like in the previous file, https://github.com/iCog-Labs-Dev/metta-moses/pull/231#discussion_r2053498594